src/collapse/dtrace.rs

```diff
@@ -1,32 +1,46 @@
 use std::collections::VecDeque;
-use std::io;
-use std::io::prelude::*;
+use std::io::{self, prelude::*};
 
-use fnv::FnvHashMap;
 use log::warn;
 
-use super::Collapse;
+use super::{Collapse, Input, Occurrences, CAPACITY_INPUT_BUFFER, CAPACITY_LINE_BUFFER};
+
+///////////////////////////////////////////////////////////////////////////////
 
 /// Settings that change how frames are named from the incoming stack traces.
 ///
-/// All options default to off.
-#[derive(Clone, Debug, Default)]
+/// All options default to off, expect nthreads, which defaults to the number
+/// of logical cores on your machine.
+#[derive(Copy, Clone, Debug)]
```

Comment:
I think I'd leave off `Copy`, since it would make adding something like a `String` to these `Options` a backwards-incompatible change.

Reply:
👍